### DISCUSSION TITLE
【中文翻译】Danbooru.csv 分享

### DISCUSSION THREAD

**Discussion by HalfMAI** [DominikDoom/a1111-sd-webui-tagcomplete#23] [Show and tell]
使用 Google Sheet 文档的功能进行了对 Danbooru.csv 的类型0，进行了中文翻译，对部分（大概前面1k多条进行了简单的校正）
同时使用了来自于 nga@猫舌头的阿巧 对于部分tag 的翻译进行替换，以保证他翻译的更优先
https://ngabbs.com/read.php?tid=33869519&rand=887

这里提供两种类型文件下载：

1.只包含 nga@猫舌头的阿巧 的文件
https://gist.github.com/HalfMAI/e20a974a8b87bbb63d8da8051442b6b2
2.完整的danbooru.csv 的 类型0 的翻译文件（包含 nga@猫舌头的阿巧  翻译部分）
[danbooru-0-zh.csv](https://github.com/DominikDoom/a1111-sd-webui-tagcomplete/files/9819578/danbooru-0-zh.csv)

使用方法：
1. 放置下载的文件于 **tags** 文件夹
![image](https://user-images.githubusercontent.com/5293796/196667725-fd73b70e-44f7-40a9-a3db-d5d0e5ada2ef.png)

2. 修改 **config.json** 中的 extraFile 字段文件名字
![image](https://user-images.githubusercontent.com/5293796/196667452-696dd9d2-9bcb-49c7-ae00-2bd855ab5230.png)

3. 重新启动即可

**Comment by idk500** [comment]
感谢分享!

**Comment by QR-0W** [comment]
只要使用这个CSV，WebUI就会失去响应，求解

**Reply by DominikDoom** [reply]
注意：我不懂中文，这是机器翻译。

这已经很老了，Extra file不应该再被用来做这个。有一个新的Translation file选项用于此目的，没有这个问题，见 #120 中的截图和解释

**Reply by QR-0W** [reply]
> 注意：我不懂中文，这是机器翻译。
> 
> 这已经很老了，Extra file不应该再被用来做这个。有一个新的Translation file选项用于此目的，没有这个问题，见 #120 中的截图和解释

In fact, the problem manifests itself in that as soon as I put this CSV file with Chinese translation into the tags folder, the whole WebUI loses its response. And unless I disable the entire extension, this problem, even after I move that CSV out of the folder, will continue to happen

**Reply by DominikDoom** [reply]
That means the file is still getting loaded. Have you reset the extra file setting back to "None"? You can also do that without the webui by locally editing `config.json` in the webui root and changing the line to `"tac_extra.extraFile": "None",` if it is stuck loading.

**Reply by QR-0W** [reply]
> That means the file is still getting loaded. Have you reset the extra file setting back to "None"? You can also do that without the webui by locally editing `config.json` in the webui root and changing the line to `"tac_extra.extraFile": "None",` if it is stuck loading.

![图片](https://user-images.githubusercontent.com/84510337/218259616-0f336741-8ca5-493a-8cc4-519df0fe92c1.png)
i'm trying to use the extension without Chinese translation, and it works well. Now I will test how it works with Chinese translation.

**Reply by QR-0W** [reply]
> That means the file is still getting loaded. Have you reset the extra file setting back to "None"? You can also do that without the webui by locally editing `config.json` in the webui root and changing the line to `"tac_extra.extraFile": "None",` if it is stuck loading.

so, now it started working again after I tried to uninstall the extension and install the latest release version. And another problem is that i can't find a such thing as `config.json` in the `tags` folder.

**Reply by DominikDoom** [reply]
I updated the extension to use the normal settings page in the webui quite a while ago, so it doesn't use a separate config.json file anymore.
Everything can be either changed in the settings tab or by editing the webui's config.json file in the root folder used to store *all* settings.

**Reply by QR-0W** [reply]
All right, thanks a lot!
